app/views/projects/new.html.erb

```diff
@@ -15,11 +13,24 @@
     <%= f.label :target_amount%>
     <%= f.text_field :target_amount%>
 
-    <%= f.label :completion_date%>
-    <%= f.text_field :completion_date%>
+    <%= f.label :category %>
```

Comment:
Had to move all of the dropdown functions to the bottom of the form in order to keep them from hiding other label fields